File: frontend/src/app/models/market-status.ts
//
// Date: 9/8/2017
// Author(s): Spicer Matthews (spicer@options.cafe)
// Copyright: 2017 Cloudmanic Labs, LLC. All rights reserved.
//

export class MarketStatus {
  
  //
  // Construct.
  //
  constructor(
    public description: string,
    public state: string
  ){}
  
  //
  // Build object for emitting to the app.
  //
  public static buildForEmit(data) : MarketStatus  {
    return new MarketStatus(data.State, data.Description); 
  }
}

/* End Files */

File: backend/library/state/market_status.go
//
// Date: 2/26/2017
// Author(s): Spicer Matthews (spicer@options.cafe)
// Copyright: 2017 Cloudmanic Labs, LLC. All rights reserved.
//

package state

var (
	marketStatus interface{}
)

//
// Set market status.
//
func SetMarketStatus(status interface{}) {
	marketStatus = status
}

//
// Get market status.
//
func GetMarketStatus() interface{} {
	return marketStatus
}

/* End File */
